frontend/js/main.js

```diff
@@ -0,0 +1,1910 @@
+  rawFile.onreadystatechange = function () {
+    if (rawFile.readyState == 4 && rawFile.status == "200") {
+      callback(rawFile.responseText);
+    }
+  }
+  rawFile.send(null);
+}
+
+/**
+ * Function to create session table content
+ *
+ * @return {null}
+ */
+const setSessionTableContent = () => {
+  let sessionArray = JSON.parse(sessionStorage.getItem('sessionStorage'));
+  let sessionContent = '';
+  if (sessionArray === null) {
+      sessionContent = `<tr>
+                          <td colspan='5' class='center session-image'><img src='Icons/Icons/Group 2154.svg' alt='nothing to show'></td>
+                        </tr>
+                        <tr>
+                          <td colspan='5' class='center simple-grey-text'>No active session available! <br> Please connect a database to initiate a new session.</td>
+                        </tr>`;
+  }
+  else {
+    let sessionArrayLength = sessionArray.length;
+    for (var x = 0; x < sessionArrayLength; x++) {
+      let session = sessionArray[x];
+      let sessionName = session.fileName;
+      let sessionDate = session.createdAt.substr(0, session.createdAt.indexOf("T"));
+      let sessionTime = session.createdAt.substr(session.createdAt.indexOf("T") + 1);
+      sessionContent += `<tr class='d-flex'>
+                          <td class='col-2 session-table-td2'>${sessionName}</td>
+                          <td class='col-4 session-table-td2'>${sessionDate}</td>
+                          <td class='col-2 session-table-td2'>${sessionTime}</td>
+                          <td class='col-4 session-table-td2 session-action' id=${x}>
+                            <a href='#/schema-report-resume-session' style='cursor: pointer; text-decoration: none;' onclick='resumeSessionHandler(${x}, ${JSON.stringify(sessionArray)})'>Resume Session</a>
+                          </td>
+                        </tr>`;
+    }
+  }
+  return sessionContent;
+}
+
+/**
+ * Function to handle resume session click event
+ *
+ * @param {number} index session index in the array
+ * @param {array} sessionArray array of objects containing session information
+ * @return {null}
+ */
+const resumeSessionHandler = (index, sessionArray) => {
+  storeResumeSessionId(sessionArray[index].driver, sessionArray[index].path, sessionArray[index].fileName, sessionArray[index].sourceDbType);
+}
+
+/**
+ * Function to check source schema while importing any file
+ *
+ * @param {string} val source db value (mysql or postgres)
+ * @return {null}
+ */
+const importSourceSchema = (val) => {
+  let sourceTableFlag = '';
+  if (val === 'mysql') {
+    sourceTableFlag = 'MySQL';
+    localStorage.setItem('sourceDbName', sourceTableFlag);
+  }
+  else if (val === 'postgres') {
+    sourceTableFlag = 'Postgres';
+    localStorage.setItem('sourceDbName', sourceTableFlag);
+  }
+}
+
+/**
+ * Function to get paths and events generated from window
+ *
+ * @param {object} params object containing path and event as keys
+ * @return {null}
```

Comment:
Uhhh this returns a boolean right?

Reply:
Yes, it is now modified to boolean return type